narratives/ncov_sit-rep_2020-03-27.md

```diff
@@ -0,0 +1,2734 @@
+* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=3).  
+* [Updates for Europe](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=4).  
+* [Updates for North America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=7).  
+* [Updates for Central & South America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=11).  
+* [Updates for Asia](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=13).  
+* [Updates for Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=15).  
+* [Updates for Oceania](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=17).  
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=19).  
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=20).  
+<!-- This is right-side text -->
+```auspiceMainDisplayMarkdown
+# Executive summary
+
+We analyzed 1,495 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is evolving and moving around the world. In this week's updates, we report:
+
+**Europe**  
+* We continue to see co-mingling of samples across Europe, suggesting that the virus has continued to move across borders in the last 3-5 weeks. As mitigation measures have time to take effect, we may hope to see more clustering of cases by country.
+* The North American and European outbreaks are deeply intertwined, with likely transmission in both directions.  
+
+**North America**   
+* Within the U.S., transmission patterns are complex, with samples collected from opposite sides of the country still showing close relationships.
+* Washington state has had at least two independent introductions (the first likely from China, the second likely from Europe), which have led to two separate transmission chains.
+* Evidence for likely local transmission within several states, most clearly within California.
+
+**Central and South America**
+* We received new sequences from South America this week, but sampling remains sparse; there may be more COVID activity in this region than is captured in the available data, but we can't say for sure.
+
+**Asia**  
+* Preliminary evidence for a possible second transmission chain circulating within Iran.
+
+**Africa**  
+* Evidence for likely local transmission within Kinshasa, Democratic Republic of the Congo, for the last 11+ days.
+* We received new sequences from Africa this week, but sampling remains sparse; there may be much more COVID activity in this region than is captured in the available data.
+
+**Oceania**
+* Evidence for likely local transmission in New South Wales, Australia.
+```
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 2-->
+# [COVID-19 Resources](https://nextstrain.org/ncov/2020-03-27)
+We've prepared some resources that are worth reading to familiarize yourself with COVID-19 and the virus that causes it, SARS-CoV-2.
```

Comment:
```suggestion
We've prepared some resources that are worth reading to familiarize yourself with COVID-19 and the virus that causes it, SARS-CoV-2, also known as hCov-19.
```
Since we use hCov-19 in the text opposite. It might be adding this.